### PR TITLE
User guide update

### LINKED ISSUE #176: [PE-D] When removing inventory > current quantity, no error is shown
No idea if it is intentional, but I am able to remove more than the current quantity in the inventory. Theres no error message, and shows that the excess number is removed and not the original number

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: foochifa/ped#6

### LINKED ISSUE #178: [PE-D] Documentation of quanity in add ingredient
UG did not show that ingredient quantity should be the format <Integer> <String of alphabets>

Yet display box shows it.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: foochifa/ped#4

### LINKED ISSUE #180: [PE-D] Missing backtick in UserGuide
For the features section, there is a missing backtick for the second last point.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: JeremyLoh/ped#8

### LINKED ISSUE #181: [PE-D] Unable to add step in existing recipe
I am unable to add a step in an existing recipe based on the sample command. I have changed the index to 1 for both index and step index.

Executing the above command

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: JeremyLoh/ped#7

### LINKED ISSUE #183: [PE-D] Case sensitive addition of ingredients
The current inventory list has "Bacon" as one of the food items.
If I were to add "bacon", it adds it as a new item. Perhaps you could make the addition case-insensitive and when you display the items to the user, capitalize the food item.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: JeremyLoh/ped#5

### LINKED ISSUE #187: [PE-D] bug in searching function
The searching result does not match keywords. ("Pasta", "Cream" missing)

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: e0316059/ped#1

### LINKED ISSUE #188: [PE-D] The section 4 FAQ still have addressbook information.
Q: How do I transfer my data to another Computer?
A: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Address Book folder.

In the faq, the UG still mentions Adrress Book folder.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: sheexiong/ped#10

### LINKED ISSUE #189: [PE-D] In section 3.4.3 Remove ingredients from the cart, the description does not match the function and command.
In 3.4.3, the description mentions that if the quantity to be removed is not specified, all of the specified ingredient will be removed. But in the given command, it does not state that `q/QUANTITY` is optional and the command also not working if don't put the quantity.

The diagram below is the command without the quantity.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: sheexiong/ped#9

### LINKED ISSUE #194: [PE-D] The seond example in UG Recipe command section 3.2.1 `Add an ingredient to a recipe` not working.
For this bug, I found out that the issue is the command given in the UG is wrong, the correct command should be 
`recipe 2 add ingredient i/Milk q/200 ml` instead of `recipe 2 add i/Milk ingredient q/200 ml`.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: sheexiong/ped#3

### LINKED ISSUE #200: [PE-D] Possibly use code blocks for documentation
Suggestion to use `this` for your variables and commands

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Deunitato/ped#12

### PR DESCRIPTION
**Pull request by tzihiang**
Updated for the following issues:
- #200 
- #194 
- #189 
- #188 
- #187 
- #183 
- #181 
- #180 
- #178
- #176 
- #171 
- #170

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -76,10 +76,12 @@ or as `cookbook add recipe n/Cheesecake d/Delicious New York cheesecake`).
 * Items with `…`​ after them can be used multiple times. +
 (e.g. `cookbook add recipe n/NAME d/DESCRIPTION [i/INGREDIENT] [q/QUANTITY]...`
 can be used to add multiple ingredients and its respective quantities in the same command such as
-`cookbook add recipe n/Sandwich d/Delicious Sandwich i/Bread q/2 pieces i/Ham q/3 slices i/Cheese q/2 slices).
+`cookbook add recipe n/Sandwich d/Delicious Sandwich i/Bread q/2 pieces i/Ham q/3 slices i/Cheese q/2 slices`).
 * Parameters can be in any order. +
 (e.g. if the command specifies `n/NAME t/TAG i/INGREDIENT q/QUANTITY`,
 `q/QUANTITY t/TAG i/INGREDIENT` is also acceptable).
+* Commands formats given in the lowercase are case-sensitive. (eg `cookbook` is given as all lowercase. `Cookbook` or `COOKBOOK` will not work.)
```
Comment (teo-jun-xiong) [inline docs/UserGuide.adoc]: I think it is only the command word and the delimiters right (i/) right? Maybe can clarify that.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -153,6 +155,8 @@ This command allows you to remove a recipe of the specified index from the cookb
 
 ==== Search recipes by keyword
 This command allows you to search for recipes by a keyword. The search result will be returned along with the respective recipe indices.
+Adding more tags will not narrow the search results, but would expand the search results to return any recipe that contains the given tags.
```
Comment (teo-jun-xiong) [inline docs/UserGuide.adoc]: This should be keyword instead of tag?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -243,14 +248,15 @@ _(If recipe 2 specifies 500 ml of milk, it will be subtracted and updated to 300
 
 This command allows you to add a preparation step to the selected recipe.
 Requires a valid index in the list of recipes and the cooking step.
+The STEP_INDEX given must be at most 1 more than the current number of steps currently in the recipe.
```
Comment (teo-jun-xiong) [inline docs/UserGuide.adoc:271]: An example would help bring out the point here

Comment (tzihiang) [reply docs/UserGuide.adoc:271]: Thanks for looking out! Pushed a commit for all the changes for this and the above

**Review by hans555 — APPROVED** [review]

**Comment by tzihiang** [comment]
This current commit removes conflicts with PR. It prioritizes the user guide edits from the previous PR #220

**Pull request merged**